base/strings/util.jl

```diff
@@ -36,7 +36,15 @@ startswith(a::Vector{UInt8}, b::Vector{UInt8}) =
 
 # TODO: fast endswith
 
-chop(s::AbstractString) = s[1:end-1]
```

Comment:
Perhaps we still want to keep this method, since it is more efficient in the common case?